src/common/openapi/api-token-info.ts

```diff
@@ -0,0 +1,18 @@
+import { TokenInfo as DomainTokenInfo } from '../entities/token-info.entity';
+import { ApiProperty } from '@nestjs/swagger';
+import { TokenType } from '../entities/token-type.entity';
+
+export class TokenInfo implements DomainTokenInfo {
```

Comment:
I would have preferred to have `ApiTokenInfo` or similar – that would also remove the import alias `as DomainTokenInfo` (we could just use `TokenInfo`).

However it seems that `@nestjs/swagger` generates the name of the stubs based on the class name (I could not find a decorator on the class level to provide a name for it).